docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/snow-create.md

```diff
@@ -0,0 +1,379 @@
+---
+sidebar_label: snow-create
+description: Integrates with ServiceNow to manage Change Requests, Incidents, Problems, etc., and track promotion workflows.
+---
+
+# `snow-create`
+
+<span class="tag professional"></span>
+<span class="tag beta"></span>
+
+:::info
+This promotion step is only available in Kargo on the
+[Akuity Platform](https://akuity.io/akuity-platform), versions v1.9.0 and above.
+:::
+
+This page is for `snow-create` step. You can create a ServiceNow record using this step. Check below for other ServiceNow promotion steps.
+
+ServiceNow integration for kargo is a group of promotion steps:  
+1. [snow-create](./snow-create.md) 
+2. [snow-update](./snow-update.md)
+3. [snow-delete](./snow-delete.md)
+4. [snow-query-for-records](./snow-query-for-records.md)
+5. [snow-wait-for-condition](./snow-wait-for-condition.md)
+
+These steps provide a comprehensive integration with ServiceNow, allowing you to create, update, delete, query for records, and track
+promotion workflows. This is particularly useful for maintaining traceability between
+your promotion processes and project management activities.
+
+All the steps above support different record types in ServiceNow including managing Change Requests, Incidents, 
+Problems etc.,. This integration also provides condition tracking through `snow-wait-for-condition`, making it a powerful tool 
+for promotion workflows that require coordination with project management systems.
+
+`snow-create` step supports two APIs:  
+#### 1. Change Management API
+This API is used primarily for managing Change Requests.
+
+Official documentation is available [here](https://www.servicenow.com/docs/bundle/zurich-api-reference/page/integrate/inbound-rest/concept/change-management-api.html).
+
+URL format: `/api/sn_chg_rest/{api_version}/change/{change_sys_id}/task/{task_sys_id}`
+
+#### 2. Table API
+This API is used for managing Incidents, Problems, and other record types.
+
+Official documentation is available [here](https://www.servicenow.com/docs/bundle/zurich-api-reference/page/integrate/inbound-rest/concept/c_TableAPI.html).
+
+URL format: `/api/now/{api_version}/table/{tableName}/{sys_id}`
```

Comment:
This is too much for the user, just create an `info` block with one line to let the user know the step supports change management api for change requests and table api for all ticket types. and hyperlink the keywords.